tests/coalaTest.py

```diff
@@ -425,7 +425,7 @@ def test_nested_language_run_coala(self):
             os.path.dirname(__file__),
             'processes/section_executor_test_files'))
 
-        testcode_p_path = os.path.join(config_path, 'test.py')
+        testcode_p_path = os.path.join(config_path, 'test.py.txt')
```

Comment:
Is this issue somewhere reported in the main repositories?

If so can you link to the issue?
And if not, then explain how this is interfering currently with the changes? If it hasn't cause problems till now, there should be a reason for why it is a problem now.